apps/backend/src/modules/users/user.service.ts

```diff
@@ -45,6 +52,13 @@ class UserService implements Service {
 		};
 	}
 
+	public async findByEmail(
+		email: UserEntity["email"],
```

Comment:
```suggestion
		email: string,
```
Don't overcomplicate